### DISCUSSION TITLE
Text Rechts fixieren

### DISCUSSION THREAD

**Discussion by Halloka001** [olikraus/u8g2#1658] [Q&A]
Guten Abend, 
erst einmal vielen dank für deine Arbeit. Gibt es eine einfache Möglichkeit einen Text rechts am Bildschirm zu fixieren?
Wenn ich beispielsweise Zahlen hochlaufen lasse sind diese standartmäßig Linksorientiert. Bin noch Blutiger Anfänger in Sachen Arduino und Co.

**Comment by olikraus** [comment]
Am besten wäre es wohl einfach einen Font zu benutzen, der als vorletzten Buchstaben ein "m" (für monospaced). Bei den Fonts haben alle Zeichen die gleiche Breite und das hochzaehlen sieht entsprechend gut aus. Grundsätzlich wären alle Fonts aus der X11 Gruppe geeignet.

**Comment by Halloka001** [comment]
Ich glaube ich habe mich etwas unverständlich ausgedrückt. Hier in Video zu sehen Fangen die Einstelligen Zahlen Links an und erweitern sich zu zweistellig nach rechts. Meine Frage ist ob dies genau umgekehrt funktioniert. Sprich die Zahlen Fangen rechts am Rand an und erweitern sich dann nach links.
https://youtu.be/NQ-aZmnDgOM

So in etwa soll es aussehen.
https://youtu.be/mRO_A2csLRQ

Besten Dank.

**Comment by olikraus** [comment]
Hm, das könnte beispielsweise über die Funktion sprintf erreicht werden indem entsprechend viele Leerzeichen vorangestellt werden.

**Comment by Halloka001** [comment]
Danke ich werde mir das heut Abend Mal genauer anschauen.
